### PR TITLE
Nodes: Rename `remainder()` to `modInt()`.

### PR DESCRIPTION
**Pull request by cmhhelgeson**
**Description**

~~Between WGSL and GLSL, there are differences in how the commonly used modulus operation is handled. GLSL has a function mod() that computes _x modulo y_. It does not permit the use of the % operator to perform the same operation. Meanwhile, WGSL does allows the use of the % operator to compute the modulus of two numbers, and therefore has no equivalent function mod. Therefore, when the % operator is used in a TSL shader via the remainder() function, that shader will be valid for the renderer's WebGPUBackend, but invalid for its WebGLBackend. While a user could simply write their shader to use remainder or mod depending on what graphics API they are targeting, this solution negates the inherent portability of TSL shaders across shader languages.~~

Modified to simply rename remainder and mod to make their usage more clear.

### CODE REVIEW

**Comment by sunag** [comment]
`remainder()` is used for integer, and `mod()` for float. You tried replace remainder to mod() in your example? I think that rename `remainder()` to `modInt()` would be less confuse instead of change it.

**Comment by cmhhelgeson** [comment]
> `remainder()` is used for integer, and `mod()` for float. You tried replace remainder to mod() in your example? I think that rename `remainder()` to `modInt()` would be less confuse instead of change it.

Yes that fixed it. I did also try an arbitrary modulus with WebGL and that worked fine. I'm not sure why so much documentation seems to indicate that % is not allowed in GLSL at all? If it's okay, can I just change this pull request to implement the suggested name change.

**Comment by sunag** [comment]
> If it's okay, can I just change this pull request to implement the suggested name change.

Sounds great!

**Comment by cmhhelgeson** [comment]
> > If it's okay, can I just change this pull request to implement the suggested name change.
> 
> Sounds great!

Ok cool, I'll also make sure there's a deprecation option per the new standards.

**Comment by sunag** [comment]
Could you keep `mod()` name? I think rename `remainder()` to `modInt()` is better.

File: src/nodes/math/OperatorNode.js [hunk 1]
```diff
@@ -303,4 +303,14 @@ addNodeElement( 'bitXor', bitXor );
 addNodeElement( 'shiftLeft', shiftLeft );
 addNodeElement( 'shiftRight', shiftRight );
 
+
+export const remainder = ( ...params ) => {
```
Comment (sunag) [inline src/nodes/math/OperatorNode.js:307]: I think could be good add what release it was deprecated
```js
export const remainder = ( ...params ) => { // @deprecated, r168
```

**Pull request merged**